packages/ripple-nuxt-tide/modules/news/pages/index.vue

```diff
@@ -46,9 +46,13 @@ export default {
       }
     },
     publishDateAndAuthor () {
+      const hasLocation = (val) => Array.isArray(val) && val.length > 0 && val[0].hasOwnProperty('name')
+
+      const locationsArray = this.page.field_location.map(loc => loc.name)
+      const locations = locationsArray.slice(0, -1).join(', ') + ', and ' + locationsArray.slice(-1)
```

Comment:
@jlstant can you explain whats going on here? Does the last one need an `and` in between?

Reply:
So it reads "Location one, Location 2 and Location 3" ? - Ok nice thats better